packages/app-page-builder/src/editor/plugins/toolbar/onboarding/Onboarding.tsx

```diff
@@ -91,6 +97,7 @@ const Onboarding = ({ deactivatePlugin, showOnboarding }) => {
 
     return (
         <Overlay>
```

Comment:
@Fsalker just wondering if you could put the `onClick` directly on this `<Overlay>`?

Reply:
There is also no need for the `OutsideContainer` element. Everything can go into `Overlay`

Reply:
I did try that initially, but it would also cause clicking the <Container> to close the modal.

The <Overlay> seems to be all over the screen, so I just created an element behind the Container.

Reply:
Cool, that makes sense.  I'll approve the PR now.